development_extensions/oni-dev-extension/extension.js

```diff
@@ -97,17 +97,23 @@ function activate(context) {
 		// Display a message box to the user
 		vscode.window.showInformationMessage('Hello from extension!');
 	});
+    
+	let disposable5 = vscode.commands.registerCommand('developer.oni.showWorkspaceRootPath', () => {
+		// Display a message box to the user
+		vscode.window.showInformationMessage("rootPath: " + vscode.workspace.rootPath);
+	});
 
     // Helper command to show buffer text
     // This helps us create a test case to validate buffer manipulations
-	let disposable5 = vscode.commands.registerCommand('developer.oni.getBufferText', () => {
+	let disposable6 = vscode.commands.registerCommand('developer.oni.getBufferText', () => {
 		vscode.window.showInformationMessage("fulltext:" + latestText);
 	});
```

Comment:
Instead of using non-descriptive numbered identifiers, and having to renumber them when adding a new one, it might be better to write a local helper function that both does the registration  and adds the dispose function to `context.subscriptions`.

Reply:
Good suggestion, thanks!